.github/actions/install-rust/action.yml

```diff
@@ -14,8 +14,11 @@ runs:
   using: composite
   steps:
     - name: Install Rust
-      uses: dtolnay/rust-toolchain@stable
+      uses: dtolnay/rust-toolchain@1.83.0
       with:
         targets: ${{ inputs.targets }}
     - if: ${{ inputs.cache == 'true' }}
       uses: Swatinem/rust-cache@v2
+    - name: Install rustfmt
+      shell: bash
+      run: rustup component add rustfmt clippy
```

Comment:
Why is this change needed?

Reply:
it wouldn't work without it... From some reason, when specifying an explicit version, it wouldn't install these components ( which we use ). hence, I had to add them explicitly.